sound.js

```diff
@@ -158,10 +204,16 @@ Sound.prototype.setNumberOfLoops = function(value) {
 };
 
 Sound.prototype.setSpeed = function(value) {
-  this._setSpeed = value;
+  this._speed = value;
   if (this._loaded) {
-    if (!IsWindows) {
+    if (!IsWindows && !IsAndroid) {
       RNSound.setSpeed(this._key, value);
+    // Call native setSpeed method only if the media player is already playing.
```

Comment:
If these comments are located in the `else if (IsAndroid)` block, readability become to be better!